common/src/main/java/org/apache/atlas/repository/Constants.java

```diff
@@ -241,6 +241,8 @@ public final class Constants {
 
     public static final String MODIFICATION_TIMESTAMP_PROPERTY_KEY = encodePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "modificationTimestamp");
 
+    public static final String MODIFICATION_TASK_ASSET_COUNT_TO_PROPAGATE_PROPERTY_KEY = encodePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "modificationAssetsCountToPropagate");
```

Comment:
1. What is this used for?
2. Do we need the same for `assetsCountPropagated`?

Reply:
removed, not needed and not used anywhere in code